workflows/computational-chemistry/protein-ligand-complex-parameterization/protein-ligand-complex-parameterization.ga

```diff
@@ -266,7 +266,7 @@
             },
             "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/ctb_rdkit_descriptors/ctb_rdkit_descriptors/2020.03.4+galaxy1",
             "tool_shed_repository": {
-                "changeset_revision": "a1c53f0533b0",
+                "changeset_revision": "0993ac4f4a23",
```

Comment:
Wow, that looks very cool. But it seems there were no version bumps, is that correct ?

Reply:
Yes, there are none (the updated `changeset_revision`s are due to other tools in the repos being updated).

So this needs a bit more work.

Reply:
This should be fixed now, I experimented a bit more on my fork: https://github.com/simonbray/iwc/pulls.